theano/configdefaults.py

```diff
@@ -1116,6 +1116,18 @@ def is_valid_check_preallocated_output_param(param):
              # Added elsewhere in the c key only when needed.
              in_c_key=False)
 
+if not config.cxx == "gcc" and config.gcc.cxxflags:
+    _logger.warning("cxx is not set to g++. gcc.cxxflags is not effective!")
+
+AddConfigVar('icc.cxxflags',
+             "Extra compiler flags for icc",
+             StrParam(""),
+             # Added elsewhere in the c key only when needed.
+             in_c_key=False)
+
+if not config.cxx == "icpc" and config.icc.cxxflags:
+    _logger.warning("cxx is not set to icpc. gcc.cxxflags is not effective!")
```

Comment:
I just want to keep the application compatibility. If I change the gcc.cxxflags to cxx.flags and user don't know the change. The original config can't work.